### PR TITLE
updated to use sdk instead of direct api calls

### PR DESCRIPTION
**Pull request by AndresJaramillo9**
Added steps for using the Python SDK instead of making direct API calls. Also updated and added some more instructions that are more user friendly to read.

### CODE REVIEW

File: natural-language-understanding/custom_classifications_example.ipynb [hunk 1]
```diff
@@ -187,308 +211,195 @@
...
-    "print(\"Response body:\", json.dumps(response_json, indent=4, sort_keys=True), )"
-   ]
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "## Use the trained model using NLU Analyze API\n",
+    "## 5. How to Use a Trained Classifcation Model for Analysis\n",
```
Comment (evaline-ju) [inline natural-language-understanding/custom_classifications_example.ipynb:319]: spelling

**Review by kmacdonald06 — APPROVED** [review]
Everything seems right

File: natural-language-understanding/custom_classifications_example.ipynb (hunk 1 above)
Comment (kmacdonald06) [reply natural-language-understanding/custom_classifications_example.ipynb]: Fixed - thanks

**Comment by AndresJaramillo9** [comment]
Accidentally committed with my API key information exposed, the file has been updated and that key is no longer active.

File: natural-language-understanding/custom_classifications_example.ipynb [hunk 2]
```diff
@@ -187,310 +211,204 @@
...
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "## Fetch data from NLC\n",
+    "## 3. How to Train a NLU Classifications Model\n",
```
Comment (gkumbhat) [inline natural-language-understanding/custom_classifications_example.ipynb:221]: Are we removing the `Fetch data from NLC` section? We need this section actually in the notebook to be able to give customers an option to get their data from NLC.

**Review by gkumbhat — CHANGES_REQUESTED** [review]

**Review by kmacdonald06 — APPROVED** [review]
lgtm

File: natural-language-understanding/custom_classifications_example.ipynb [hunk 3]
```diff
@@ -246,251 +267,204 @@
...
-    "    response_json = response.json()\n",
+    "Use the `delete_classifications_model` method to remove a specific classification model via a `model_id`. This operation can be verified by listing all the existing models with `list_classifications_models`. "
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "source": [
+    "model_to_delete = nlu.delete_classifications_model(model_id=model_id)\n",
```
Comment (gkumbhat) [inline natural-language-understanding/custom_classifications_example.ipynb:436]: nit: would it make sense to rename this variable from `model_to_delete` to `deleted_model` ?

**Review by gkumbhat — APPROVED** [review]
Left a small nit. But otherwise looks good to me. thanks for updating the notebook.

**Comment by mediumTaj** [comment]
@kmacdonald06 Can you mere this if it looks good to you? We don't have merge permissions

**Pull request merged**